src/tempo/tempo.yaml

```diff
@@ -47,7 +46,8 @@ metrics_generator:
   ring:
     kvstore: {}
   processor:
-    service_graphs: {}
+    service_graphs:
+      dimensions: ["service.namespace"]
```

Comment:
shouldn't it be `service_namespace`?

Reply:
no, this is resource attribute name, with a dot. `.` is converted to `_` by prometheus & loki exporters later